.travis.yml

```diff
@@ -1,8 +1,41 @@
+---
 language: node_js
 node_js:
-  - "0.10.33"
-services: mongodb
+  - '0.10'
+
+sudo: required
+
+branches:
+  only:
+    - master
+    - develop
+
+git:
+  depth: 10
+
+services:
+  - mongodb
+
+cache:
+  directories:
+    - node_modules
+
 before_install:
-  - "npm install -g npm"
-  - "npm cache clean"
-  - "npm install -g adapt-cli"
+  - npm install -g npm
+  - npm install -g adapt-cli
+
+install:
+  - npm config set spin false
+  - npm install
+
+matrix:
+  fast_finish: true
+
+notifications:
+  email: false
```

Comment:
I can also confirm that I have received email notifications directly from travis on changes to the build resulting from PRs I have submitted up till now.